documentation/modules/ROOT/pages/characteristics.adoc

```diff
@@ -332,6 +332,19 @@ in the `elements` list are used for construction. When the Property using the St
 Characteristic also defines an `exampleValue`, construction and deconstruction of the value are
 tested when validating the model.
 
+[[reference-characteristic]]
+=== Reference
```

Comment:
I noted this is defined in the "Characteristics Classes" section, but it should be defined in the "Characteristics Instances" section instead.